tbconnect/tests/test_views.py

```diff
@@ -465,7 +467,25 @@ def test_reset_allowed(self):
 class TbCheckCciDataViewSetTest(APITestCase):
     url = reverse("tbcheckccidata-list")
 
+    @responses.activate
+    @override_settings(
+        CCI_AUT_URL="https://cci-data-test.com", CCI_AUT_TOKEN="test12345"
+    )
     def test_cci_data_status_code(self):
-        response = self.client.post(self.url)
+        data = {
+            "msisdn": "27821234567",
+            "name": "Tom",
+            "language": "Eng",
+            "tb_risk": "High",
+            "responded": "No",
+            "tb_tested": "Yes",
+            "tb_test_results": "Yes",
+            "screen_timeStamp": "2023-04-25 13:02:17",
+        }
+
+        responses.add(responses.POST, "https://cci-data-test.com", json=data)
```

Comment:
The json specified here is the return from CCI api, we should assert below that the api received the correct payload

Reply:
I don't need this responses as my api doesn't return data